src/io/flutter/logging/tree/CategoryCellRenderer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
nit: feel free to maintain the old copyright year when moving files like this.

Reply:
Ha!  Was just fixing. :)